spec/features/pet/pet_read_spec.rb

```diff
@@ -37,14 +37,4 @@
     expect(current_path).to eq("/shelters")
   end
 
-  it "can switch a pet's status from adoptable to pending adoption" do
```

Comment:
This tests a functionality that is no longer there, so just more sensible to get rid of it